docs/Custom_Device_Types.md

```diff
@@ -0,0 +1,184 @@
+
+### Parallel Mode
+
+In *Parallel* mode, the PCL initiates execution of models and continues to its next iteration without waiting for models to finish executing. This causes a one-cycle delay between when a model executes and when the data it produces is available to the system.
+
+1.	Get inputs from hardware devices in the system definition.
+&nbsp;&nbsp;&nbsp; **Note:** If the system includes an inline hardware interface custom device, the PCL reads DAQ digital lines and counters after the Read Data from HW case of the custom device executes in step 3.
+2.	Read asynchronous custom device FIFOs from the previous iteration.
+3.	Runs the **Read Data From HW** case of inline hardware interface custom devices.
+&nbsp;&nbsp;&nbsp; **Note:** If you configured hardware scaling, VeriStand applies the scaling after acquiring all hardware inputs.
+4.	Reads previous iteration data from models in the system definition.
+&nbsp;&nbsp;&nbsp; **Note:** This step executes on the second and subsequent iterations.
+5.	Reads data from the previous iteration of the Data Processing Loop.
+6.	Processes system mappings.
+&nbsp;&nbsp;&nbsp; **Note:** VeriStand components (including custom devices) cannot read data from a previous step until the PCL processes system mappings, even if the previous step acquired the data the component needs.
+7.	Runs the **Execute Model** case of inline model interface custom devices.
+8.	Executes steps of running real-time sequences.
+&nbsp;&nbsp;&nbsp; **Note:** VeriStand executes real-time sequences after input operations but before output operations and continues to run every step of the real-time sequence until the sequence is complete, reaches a Yield step, or completes an iteration of a loop with Auto Yield set to TRUE. If a sequence takes longer than the given time for an iteration of the PCL, the PCL runs late. To avoid errors, break up the timing of the steps by placing Yield steps throughout the sequence and enabling the Auto Yield property for any loops in the sequence.
+9.	Processes system mappings.
+10.	Writes data to models.
+11.	Initiates asynchronous execution of models.
+12.	Writes data to the Data Processing Loop.
+13.	Writes output data to hardware devices.
+14.	Runs the **Write Data to HW** case of inline hardware interface custom devices.
+15.	Writes data to asynchronous custom device FIFOs.
+
+ 
+### Low-Latency Mode
+
+In *Low* Latency mode, the PCL waits for the Model Execution Loop(s) to finish writing data to models before it reads and publishes model values to the system. This occurs during every iteration of the system. When the model completes execution, the PCL provides data from the model to other loops during the same iteration that the model generated the data.
+
+1.	Gets inputs from hardware devices in the system definition.
```

Comment:
See my comment above about footnotes, they would need to be applied to this list too.

Reply:
We kept the same format as what we saw in VeriStand Manual from ni.com:
https://www.ni.com/documentation/en/veristand/latest/manual/pcl-execution/